src/Pelagos/Util/DOIutil.php

```diff
@@ -158,6 +88,16 @@ public function createDOI(
         $status = 'reserved',
         $resourcetype = 'Dataset'
     ) {
+        if ('mint' === $doi) {
```

Comment:
I think it's better if this was just null or blank. That way you can check empty

Reply:
We don't have to check at all, if nothing is passed. No other function is going to call createDOI() with a doi parameter. We can just remove the first parameter

Reply:
> We don't have to check at all, if nothing is passed. No other function is going to call createDOI() with a doi parameter. We can just remove the first parameter

Don't we need to have a function to create existing DOI (in our system)?

Reply:
Hmm.. I don't know if there is a use case for it. Post migration to Datacite, it was useful